pkg/query-service/model/v3/v3.go

```diff
@@ -388,7 +388,7 @@ func (b *BuilderQuery) Validate() error {
 	}
 	if b.GroupBy != nil {
 		for _, groupBy := range b.GroupBy {
-			if groupBy == "" {
+			if groupBy == (AttributeKey{}) {
```

Comment:
I would add `Validate` on `AttributeKey`. The previous one simply needed to work on a string, but this changed so it should look beyond empty struct. Check if datatype is one of allowed, type etc..

Reply:
Sure